### PR TITLE
nettests: write spec for ndt7

### PR DESCRIPTION
**Pull request by bassosimone**
Closes https://github.com/ooni/probe-engine/issues/259

Closes https://github.com/ooni/probe-engine/issues/251

### CODE REVIEW

File: nettests/ts-022-ndt.md [hunk 1]
```diff
@@ -21,10 +21,24 @@ None
...
+third party server infrastructure and implementation provided by the
+[Measurement Lab](https://www.measurementlab.net/) consortium. As such,
+the [privacy policy](https://www.measurementlab.net/privacy/) of
+Measurement Lab applies on top of OONI's privacy policy.
+
+There are two flavours of the NDT protocol: ndt5 and ndt7. The ndt5
+version of the protocol is now obsolete. This documentation describes
+OONI's usage of ndt7. The specification of the ndt7 protocol is
+[available at github.com/m-lab/ndt-server](
+https://github.com/m-lab/ndt-server/blob/master/spec/ndt7-protocol.md).
```
Comment (hellais) [inline nettests/ts-022-ndt.md:36]: I wonder if we want to somehow vendor this inside of the OONI spec too.

File: nettests/ts-022-ndt.md [hunk 2]
```diff
@@ -34,12 +48,280 @@ implementation of such test used by ooniprobe.
...
 
 ## Possible conclusions
 
-TBD
+This experiment is a "here and now" snapshot of the performance between
+the user and a well provisioned reasonably-nearby server. This experiment
+does not provide information regarding the throttling of specific sites
+and/or network paths (e.g. Netflix). It may be useful, instead, to understand
+congestion. For example, preliminary data appears to clearly show the impact
+of COVID-19 lockdown on the available bandwidth in Italy.
```
Comment (hellais) [inline nettests/ts-022-ndt.md:152]: Can we put a reference to where this is shown?

Comment (bassosimone) [reply nettests/ts-022-ndt.md:152]: Maybe it's better to remove the sentence and re-add it back when we have the data

File: nettests/ts-022-ndt.md (hunk 1 above)
Comment (bassosimone) [reply nettests/ts-022-ndt.md:36]: I'm a bit worried about duplicating info, but it's also true it may be useful

Comment (bassosimone) [reply nettests/ts-022-ndt.md:36]: done in https://github.com/ooni/spec/pull/188

File: nettests/ts-022-ndt.md (hunk 2 above)
Comment (bassosimone) [reply nettests/ts-022-ndt.md:152]: done in https://github.com/ooni/spec/pull/188

**Pull request merged**